target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -24,6 +24,7 @@ ath79_setup_interfaces()
 	tplink,tl-mr3040-v2|\
 	tplink,tl-wa901nd-v2|\
 	tplink,tl-wr703n|\
+	tplink,tl-cpe-210-v3|\
```

Comment:
Your naming is not consistent (cpe210 vs. cpe-210). In addition, the "tl-" is not part of the product name here, so IMHO the best would be to use
tplink,cpe210-v3 and tplink_cpe210-v3